modules/nf-core/rtn/tna/tests/nextflow.config2

```diff
@@ -0,0 +1,5 @@
+process {
```

Comment:
I do not see this file (nor `nextflow.config3` being used).

Reply:
Instead of using the nonstandard `config{2,3,4}` notation, you could instead use Mahesh's clever system of params: https://nf-co.re/docs/guidelines/components/modules#configuration-of-extargs-in-tests